# Unbreak the autopkgtest by not bothering with python 3.8 compatibility
--- a/tempora/schedule.py
+++ b/tempora/schedule.py
@@ -3,7 +3,7 @@
 
 For example, to run a job at 08:00 every morning in 'Asia/Calcutta':
 
->>> from tests.compat.py38 import zoneinfo
+>>> import zoneinfo
 >>> job = lambda: print("time is now", datetime.datetime())
 >>> time = datetime.time(8, tzinfo=zoneinfo.ZoneInfo('Asia/Calcutta'))
 >>> cmd = PeriodicCommandFixedDelay.daily_at(time, job)
